main.go

```diff
@@ -5,35 +5,36 @@ import (
 	"os"
 
 	"github.com/jpillora/opts"
-	"github.com/jpillora/opts-cli/internal/initopts"
+	initPkg "github.com/jpillora/opts-cli/internal/init"
 )
 
 var (
-	Version string = "dev"
-	Date    string = "na"
-	Commit  string = "na"
+	version = "dev"
```

Comment:
I don't think private var can get set by the linked -X flag.

Reply:
Can indeed set private